specification/_types/aggregations/AggregationContainer.ts

```diff
@@ -427,6 +428,15 @@ export class AggregationContainer {
    * @doc_id search-aggregations-bucket-reverse-nested-aggregation
    */
   reverse_nested?: ReverseNestedAggregation
+  /**
+   *
+   * A single bucket aggregation that randomly includes documents in the aggregated results.
+   * Sampling provides significant speed improvement at the cost of accuracy.
+   * @doc_id search-aggregations-pipeline-random-sampler-aggregation
```

Comment:
```suggestion
   * @doc_id search-aggregations-bucket-random-sampler-aggregation
```

Reply:
Thanks! I removed `bucket` too because it's not in the Elasticsearch URL for the random sampler aggregation:

* Random sampler: https://www.elastic.co/guide/en/elasticsearch/reference/current/search-aggregations-random-sampler-aggregation.html
* Sampler: https://www.elastic.co/guide/en/elasticsearch/reference/current/search-aggregations-bucket-sampler-aggregation.html

Done in [919eb559 (#2944)](https://github.com/elastic/elasticsearch-specification/pull/2944/commits/919eb559939ea8cf1e421e80a4e0ff6991422b8b).